ccpp/data/CCPP_typedefs.F90

```diff
@@ -452,10 +452,18 @@ module CCPP_typedefs
      integer                             :: ie
      integer                             :: isd
      integer                             :: ied
+     integer                             :: isc1
```

Comment:
These changes are to address undiscovered bugs in GFDL MP fast physics. The allocation never matched the metadata, but we got away with it because we used assumed-shape arrays everywhere.